data/human/culture free worlds.txt

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2015 by Michael Zahniser
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# "Missions" which add culture events to the spaceport.
+
+mission "South: The Great Pig War"
+	landing
```

Comment:
Why `landing`? I'd have it minor and still in the spaceport, as it starts off talking about the spaceport.

Reply:
@Galaucus has some funny ideas. :P

Reply:
that doesn't answer my question :james:

like I told WarlordMike recently, nobody here reviewing cares about spoilers, spell out what ideas are, get feedback now, see if that changes my mind or if I still think it makes more sense if it's a spaceport-triggered mission

Reply:
Since you missed it, - I tagged him because all your questions are on his content, I can't answer them.

Reply:
I see, comment above lead me to believe you knew what the `funny ideas` were.